packages/flutter_test/test/finders_test.dart

```diff
@@ -0,0 +1,36 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/widgets.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  group('hitTestable', () {
+    testWidgets('excludes non-hit-testable widgets', (WidgetTester tester) async {
+      await tester.pumpWidget(
+        new IndexedStack(
+          sizing: StackFit.expand,
+          children: <Widget>[
+            new GestureDetector(
+              key: const ValueKey<int>(0),
+              behavior: HitTestBehavior.opaque,
+              onTap: () {},
```

Comment:
nit: `{ }` for an empty block, we use `{}` to mean an empty map.

Reply:
Done.

Reply:
looks like you missed one :-)

Reply:
D'oh! Fixed.